packages/compile-common/src/profiler/isExplicitlyRelative.ts

```diff
@@ -1,3 +1,3 @@
 export function isExplicitlyRelative(importPath: string): boolean {
-  return importPath.startsWith(".");
+  return importPath.startsWith("./") || importPath.startsWith("../");
```

Comment:
Will there ever be a valid relative path that looks like `"folder/to/thing"`?

Reply:
The Solidity documentation says:

> All path names are treated as absolute paths unless they start with the current `.` or the parent directory `..`.

So, in Solidity, this is not possible.  In Vyper, however, it is possible, which is why I had to change where this is checked.

Note that the change to this line of code doesn't relate to that though -- this change was just me fixing a bug I caught (since e.g. `".hiddenfolder/to/thing"` shouldn't be treated as a relative path).